settings/src/components/settings.js

```diff
@@ -38,8 +38,16 @@ export default function Settings() {
 				} }
 			/>
 		),
+		webauthn: (
+			<WebAuthn
+				onKeyAdd={ () => {
+					if ( ! backupCodesEnabled ) {
+						navigateToScreen( 'backup-codes' );
+					}
```

Comment:
Should we navigate to home if backup codes are already enabled, like TOTP above?
```suggestion
					if ( ! backupCodesEnabled ) {
						navigateToScreen( 'backup-codes' );
					} else {
						navigateToScreen( 'home' );
					}
```

Reply:
Personal pref: I would keep both of these DRYer by using a ternary for the screen name

```
<WebAuthn onKeyAdd={ () => navigateToScreen( backupCodesEnabled ? 'home' : 'backup-codes' ) } />
```

Reply:
> Personal pref: I would keep both of these DRYer by using a ternary for the screen name

I would typically prefer explicitness (within reason) over the use of ternaries for conciseness, in this case. It's easier to extend and debug.

> Should we navigate to home if backup codes are already enabled, like TOTP above?

I don't think so since we have a useful interface to show the user after adding a key (key list) and users can add multiple keys, whereas with TOTP there really isn't anything left to do.

Reply:
Makes sense 👍